### PR TITLE
[Improment]Add workload group resource usage metrics

### PR DESCRIPTION
**Pull request by wangbo**
### What problem does this PR solve?
Add workload group resource usage metrics, Workload Group's cpu/memory/io usage couldbe exported by metric http api.

1. metric api
```
curl http://127.0.0.1:8240/metrics


doris_be_workload_group_mem_used_bytes{name="normal"} 2213985029
doris_be_workload_group_local_scan_bytes_local_data_dir_0{name="normal"} 3934180612
doris_be_workload_group_cpu_time_sec{name="normal"} 127
doris_be_workload_group_local_scan_bytes{name="normal"} 3934329567
doris_be_workload_group_local_scan_bytes_spill_data_dir_0{name="normal"} 0
doris_be_workload_group_remote_scan_bytes{name="normal"} 0

doris_be_host_cpu_num 96
```

2 system table
```
mysql [information_schema]>select * from workload_group_resource_usage where WORKLOAD_GROUP_ID = 1;
+-------+-------------------+--------------------+-------------------+-----------------------------+------------------------------+
| BE_ID | WORKLOAD_GROUP_ID | MEMORY_USAGE_BYTES | CPU_USAGE_PERCENT | LOCAL_SCAN_BYTES_PER_SECOND | REMOTE_SCAN_BYTES_PER_SECOND |
+-------+-------------------+--------------------+-------------------+-----------------------------+------------------------------+
| 10005 |                 1 |         1668045215 |               5.2 |                   146696933 |                            0 |
+-------+-------------------+--------------------+-------------------+-----------------------------+------------------------------+
1 row in set (0.06 sec)
```

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [x] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [x] Yes. 

- Does this need documentation?
    - [ ] No.
    - [x] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

File: be/src/common/config.cpp [hunk 1]
```diff
@@ -1211,6 +1211,8 @@ DEFINE_mString(doris_cgroup_cpu_path, "");
 DEFINE_mBool(enable_be_proc_monitor, "false");
 DEFINE_mInt32(be_proc_monitor_interval_ms, "10000");
 
+DEFINE_Int32(workload_group_metrics_interval_ms, "10000");
```
Comment (yiguolei) [inline be/src/common/config.cpp:1214]: It is too long, change to 3 secs

File: be/src/runtime/workload_group/workload_group_metrics.cpp [hunk 2]
```diff
@@ -0,0 +1,126 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
```
Comment (yiguolei) [inline be/src/runtime/workload_group/workload_group_metrics.cpp:3]: 这种新文件，直接把 compile checks 加上

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**